test.requirements.txt

```diff
@@ -12,6 +12,8 @@ aiohttp-swagger==1.0.14
 pyyaml==5.1.2
 boto3==1.10.28
 imageio==2.6.1
+# PNG plugin is broken in Pillow 7.1.x
+Pillow==7.0.0
```

Comment:
<7.1? maybe?

Reply:
With such frequency of upgrade failures for widely used Python libraries I somewhat miss `package.json.lock` in NodeJS and thus I use `==` everywhere.

Reply:
:ok: